dbus/methods.go

```diff
@@ -473,6 +473,84 @@ type DisableUnitFileChange struct {
 	Destination string // Destination of the symlink
 }
 
+// MaskUnitFiles() may be used to mask one or more units in the system
+//
+// It takes a list of unit files to mask (either just file names or full
+// absolute paths if the unit files are residing outside the usual unit
+// search paths), and two booleans:
+// runtime: to specify whether the unit was enabled for runtime only
+// (true, /run), or persistently (false, /etc).
+// and force
```

Comment:
The list of parameters is a bit awkward to read (it took me a couple of seconds to understand the dangling "and force"). Can you reword the last paragraph as a bullet list like:
```
It takes three arguments:
 * list of unit files to mask, either...
 * runtime flag to specify...
 * force flag
```